src/test/java/org/openmrs/contrib/qaframework/automation/Steps.java

```diff
@@ -19,7 +19,7 @@
 
 // Use english locale, of not set, the test instance should be set to english language
 public class Steps extends ReferenceApplicationTestBase {
-	
+
```

Comment:
After testing the workflow, git identified this minor change as needed which was introduced by clinical visit test when an empty line was introduced to between Steps class declaration and variables. However, the line contained spaces and have eliminated the spaces so that the line is completely empty.